src/transformers/integrations/peft.py

```diff
@@ -292,11 +292,12 @@ def set_adapter(self, adapter_name: Union[List[str], str]) -> None:
             )
 
         from peft.tuners.tuners_utils import BaseTunerLayer
+        from peft.utils import ModulesToSaveWrapper
```

Comment:
Which version of peft is this class available from? If it's dev, then we need to make sure to safely import this

Reply:
`ModulesToSaveWrapper` has been in PEFT since before 0.5.0 which is the required version for using PEFT + transformers

Reply:
OK :)